### DISCUSSION TITLE
Jetpack Compose: Button Action & Intent

### DISCUSSION THREAD

**Discussion by algokelvin-373** [algokelvin-373/ProjectAppAndroid#67] [Ideas]
Hi, @ibnu009 and @RachmanForniandi 
I will open discussion about simple project using Jetpack Compose.
I have made 2 projects this. You can check this:

1. [Jetpack Compsose Activity](https://github.com/kelvin373-ht/ProjectAppAndroid/tree/release/v1.16.4/AndroidJetpackPro/AndroidJetpackComposeActivity)
2. [Jetpack Compose One](https://github.com/kelvin373-ht/ProjectAppAndroid/tree/release/v1.16.4/AndroidJetpackPro/JetpackComposeOne)

**Jetpack Compose Activity**
This is first project to show about modelling Activity in Jetpack Compose's mode. So, you can use this project to make new projects.

**Jetpack Compose One**
This is first project Jetpack Compose basically. This is project to show about this github "Algokelvin". This project is using TextView and Image. So, you can learn this. If you want to ask, you can comments in discussions. I will be reply you.

Thank you :)

**Comment by algokelvin-373** [comment]
Hi, @ibnu009 and @RachmanForniandi
have you ever learned Android Jetpack Compose?

If Yes, give me one idea to make project Android Jetpack Compose.

Thank you 👍

**Reply by ibnu009** [reply]
No i have not,

Perhaps we can practice pagination on compose?

**Reply by algokelvin-373** [reply]
Yes @ibnu009 
if you have some idea example pagination, go ahead.
I will look your program and discuss together.
